lib/gitlab/ci/templates/pipeline/DockerPipeline.yml

```diff
@@ -62,3 +63,11 @@ kics-iac-sast:
       sast: gl-sast-report.json
     paths:
       - gl-sast-report.json
+
+multiarch_manifest_publish:
```

Comment:
I'd leave the multiarch_manifest_publish out of the DockerPipeline.yml since not everyone is going to want to use the multiarch_manifest_publish. When people want to use the multiarch_manifest_publish in their own Gitlab pipeline they should include the multiarch_manifest_publish job directly using:

```
include: 
- remote: https://raw.githubusercontent.com/chesapeaketechnology/gitlab-templates/refs/heads/release/4/lib/gitlab/ci/templates/jobs/docker/multiarch_manifest_publish.yml
```